### PR TITLE
Fix NullPointerException in SCEntityListener

### PR DESCRIPTION
**Pull request by RodneyMKay**
Fixes a null pointer exception that occured on ny server.

### CODE REVIEW

File: src/main/java/net/sacredlabyrinth/phaed/simpleclans/listeners/SCEntityListener.java [hunk 1]
```diff
@@ -208,7 +208,7 @@ public void onPlayerInteract(PlayerInteractEntityEvent event)
                 {
                 	return;
                 }
-                if (cp.getClan().isMember((Player) tamed.getOwner()))
+                if (cp.getClan() != null && cp.getClan().isMember((Player) tamed.getOwner()))
```
Comment (RoinujNosde) [inline src/main/java/net/sacredlabyrinth/phaed/simpleclans/listeners/SCEntityListener.java:211]: Did you test this fix?
getClan wouldn't return null on a ClanPlayer retrieved by getClanPlayer

Comment (RoinujNosde) [reply src/main/java/net/sacredlabyrinth/phaed/simpleclans/listeners/SCEntityListener.java:211]: "cp" is probably the cause of the NPE

Comment (RodneyMKay) [reply src/main/java/net/sacredlabyrinth/phaed/simpleclans/listeners/SCEntityListener.java:211]: Oh, sorry about that. I did test it, but not that thuroughly. It seemed like it was an intermittent issue, but aparently it only triggers if the player is not trusted on the grief prevention claim.

Comment (RoinujNosde) [reply src/main/java/net/sacredlabyrinth/phaed/simpleclans/listeners/SCEntityListener.java:211]: > not trusted on the grief prevention claim.

GP probably cancels the event if they are trusted.

Anyway, the null check should be on "cp".

**Pull request closed without merging**